ur_robot_driver/doc/usage.rst

```diff
@@ -206,7 +206,9 @@ Custom URScript commands
 ------------------------
 
 The driver's package contains a ``urscript_interface`` node that allows sending URScript snippets
-directly to the robot. It gets started in the driver's launchfiles by default. To use it, simply
+directly to the robot when the robot is in headless/Remote control mode.
```

Comment:
Wether or not we use headless mode doesn't really matter. But yes, we should state more clearly, that the robot will have to be in remote control mode for a PS5 robot.

```suggestion
directly to the robot when the robot is in remote control mode.
```

Reply:
True it the remote control that makes it accepting code from external on Polyscope5